portfolio/src/main/webapp/src/com.js

```diff
@@ -113,17 +158,57 @@ function addCommentsToPage(comments) {
   // Clear existing HTML
   document.getElementById('comment-list').innerHTML = '';
   comments.forEach((comment) => {
-    const newCard = createCommentCard(comment);
+    const newCard = createCommentCard(
+      comment.commentText,
+      comment.sentimentScoreFlag ? comment.sentimentScore : undefined,
+    );
     document.getElementById('comment-list').appendChild(newCard);
   });
 }
 
+/**
+ * Converts sentiment score in the interval
+ * [-1.0, 1.0] to an appropriate emoji.
+ * (Note = -1.0 is extremely negative,
+ *  1.0 is extremely positive). Numbers outside
+ * this interval will be rounded towards zero
+ * to either -1.0 or 1.0.
+ *
+ * @param {number} sentimentScore
+ * @return {String} A string containing a single
+ * emoji character.
+ */
+function convertSentimentScoreToEmoji(sentimentScore) {
+  // Emotions are scaled to 11 different emojis.
```

Comment:
Why `11` and not `10`? I'm curious since there are two of the same in the middle.

Reply:
The 11 emojis represent each of the following intervals:
[-1.0, -0.8)
[-0.8, -0.6)
. . .
[0.8, 1.0)
and {1.0}.

I will change it such that the final interval includes 1.0 in this PR.